src/utils/image_server.cljs

```diff
@@ -5,6 +5,8 @@
 (def ^:const identicons-action "/messages/identicons")
 (def ^:const account-images-action "/accountImages")
 (def ^:const contact-images-action "/contactImages")
+(def ^:const generate-qr-action "/GenerateQRCode")
```

Comment:
I understand other endpoints in the HTTP server have a similar problem, but I think the URI hostname and path should avoid mixing uppercase and lowercase characters, they should be just lowercase. And words in the hostname and path are usually separated by dashes or underscores, not camel case.

I don't have references now, but I think it's a general good practice.

Reply:
Thats something we should address at status-go side, filed an issue for it : https://github.com/status-im/status-go/issues/3434